src/graphql/customer.rs

```diff
@@ -577,6 +577,134 @@ pub async fn broadcast_customer_networks(
+            .execute(
+                r#"mutation {
+                insertCustomer(name: "t2", description: "", networks: [])
+            }"#,
+            )
+            .await;
+        assert_eq!(res.data.to_string(), r#"{insertCustomer: "1"}"#);
+
+        let res = schema
+            .execute(
+                r#"mutation {
+                insertCustomer(name: "t3", description: "", networks: [])
+            }"#,
+            )
+            .await;
+        assert_eq!(res.data.to_string(), r#"{insertCustomer: "2"}"#);
+
+        let res = schema
+            .execute(
+                r#"mutation {
+                insertCustomer(name: "t4", description: "", networks: [])
+            }"#,
+            )
+            .await;
+        assert_eq!(res.data.to_string(), r#"{insertCustomer: "3"}"#);
+
+        let res = schema
+            .execute(
+                r#"mutation {
+                insertCustomer(name: "t5", description: "", networks: [])
+            }"#,
+            )
+            .await;
+        assert_eq!(res.data.to_string(), r#"{insertCustomer: "4"}"#);
+
+        let res = schema
+            .execute(
+                r#"mutation {
+                insertCustomer(name: "t6", description: "", networks: [])
+            }"#,
+            )
+            .await;
+        assert_eq!(res.data.to_string(), r#"{insertCustomer: "5"}"#);
+
+        let res = schema
+            .execute(
+                r#"mutation {
+                insertCustomer(name: "t7", description: "", networks: [])
+            }"#,
+            )
+            .await;
+        assert_eq!(res.data.to_string(), r#"{insertCustomer: "6"}"#);
+
+        let res = schema
+            .execute(
+                r#"mutation {
+                insertCustomer(name: "t8", description: "", networks: [])
+            }"#,
+            )
+            .await;
+        assert_eq!(res.data.to_string(), r#"{insertCustomer: "7"}"#);
+
+        let res = schema
+            .execute(
+                r#"mutation {
+                insertCustomer(name: "t9", description: "", networks: [])
+            }"#,
+            )
+            .await;
+        assert_eq!(res.data.to_string(), r#"{insertCustomer: "8"}"#);
+
+        let res = schema
+            .execute(
+                r#"mutation {
+                insertCustomer(name: "t10", description: "", networks: [])
+            }"#,
+            )
+            .await;
+        assert_eq!(res.data.to_string(), r#"{insertCustomer: "9"}"#);
+
+        let res = schema
+            .execute(r#"{customerList(last:10){edges{node{name}}totalCount}}"#)
+            .await;
+        assert_eq!(
+            res.data.to_string(),
+            r#"{customerList: {edges: [{node: {name: "t1"}},{node: {name: "t10"}},{node: {name: "t2"}},{node: {name: "t3"}},{node: {name: "t4"}},{node: {name: "t5"}},{node: {name: "t6"}},{node: {name: "t7"}},{node: {name: "t8"}},{node: {name: "t9"}}],totalCount: 10}}"#
+        );
+
```

Comment:
let's add cases where
- last:10 && before: "t8"
- last:10 && after: "t3"
- first:10
- first: 10 && before: "t8"
- first:10 && after: "t3"

and check that order of node stays consistent, for all the cases.

Reply:
Sounds good! Once `customer` is done I'm going to use what I did in `customer` to the other APIs.

Reply:
`account_list` & `node_status_list` seem good

Reply:
@sophie-cluml Made an issue for this and will do this in a separate PR